@remirror/extension-tables/src/types.ts

```diff
@@ -0,0 +1,5 @@
+import { NodeExtensionSpec } from '@remirror/core';
```

Comment:
Maybe rename the file to `table-types.ts` to make it easier for people editing in the future.

There are already a lot of `types.ts` files which need to be refactored to better names.